client/CPlayerInterface.cpp

```diff
@@ -2582,6 +2582,9 @@ void CPlayerInterface::artifactMoved(const ArtifactLocation &src, const Artifact
 		if (artWin)
 			artWin->artifactMoved(src, dst);
 	}
+
+	GH.totalRedraw();

```

Comment:
what for is it? does it impact performance?

Reply:
If that's not there, the transferred artifacts don't get displayed unless the window is closed and reopened.

Reply:
I'll add a comment